dev-docs/bidders/exadsadserver.md

```diff
@@ -0,0 +1,487 @@
+| `bidfloorcur`      |  |  Currency for minimum bid value specified using ISO-4217 alpha codes           | `'EUR'`    | `'string'` |
+| `bcat`      |  |  Blocked advertiser categories using the IAB content categories           | `['IAB25', 'IAB7-39','IAB8-18','IAB8-5','IAB9-9']`    | `'string array'` |
+| `badv`      |  |  Block list of advertisers by their domains            | `['first.com', 'second.com']`    | `'string array'` |
+| `mimes`      |  |   List of supported mime types. We support: image/jpeg, image/jpg, image/png, image/png, image/gif, image/webp, video/mp4            | `['image/jpg']`    | `'string array'` |
+| `dsa`      |  |   DSA transparency information. To see the specific paragraph for more details.            | `{ dsarequired: 3, pubrender: 0, datatopub: 2 }`    | `object` |
+| `endpoint`      | required |   EXADS endpoint (URL)            | `https://your-ad-network.com/rtb.php`    | `string` |
+
+##### DSA
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Description           | Example   | Type      |
+|---------------|----------|-----------------------|-----------|-----------|
+| `dsarequired`      |  |  flag to indicate if DSA information should be made available          | `3`    | `'integer'` |
+| `pubrender`      |  | flag to indicate if the publisher will render the DSA Transparency info          | `0`    | `'integer'` |
+| `datatopub`      |  | ndependent of pubrender, the publisher may need the transparency data for audit purposes           | `2`    | `'integer'` |
+
+##### RTB Banner 2.4 (Image)
+
+```javascript
+
+adUnits = 
+    [{  code: 'postbid_iframe', // the frame where to render the creative
+        mediaTypes: {
+            banner: {
+                sizes: [300, 250]
+            }
+        },
+        bids: [{
+            bidder: 'exadsadserver',
+            params: {
+                    zoneId: 12345,
+                    fid: '829a896f011475d50da0d82cfdd1af8d9cdb07ff',
+                    partner: 'ortb_2_4',
+                    siteId: '123',
+                    siteName: 'test.com',
+                    userIp: '0.0.0.0',
+                    userId: '1234',
+                    country: 'IRL',
+                    impressionId: impression_id.toString(),
+                    keywords: 'lifestyle, humour',
+                    bidfloor: 0.00000011,
+                    bidfloorcur: 'EUR',
+                    bcat: ['IAB25', 'IAB7-39','IAB8-18','IAB8-5','IAB9-9'],
+                    badv: ['first.com', 'second.com'],
+                    mimes: ['image/jpg'],
+                    dsa: {
+                        dsarequired: 3, 
+                        pubrender: 0,
+                        datatopub: 2
+                    },
+                    endpoint: 'https://your-ad-network.com/rtb.php'
+                }
+            }]
+        }];
+```
```

Comment:
@giuseppe-exads In these snippets, we should probably use standard Javascript formatting and indentation (c.f., Prettier defaults).

I would also add `// optional` to the parameters that aren't required like in:
https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/openx.md#adunit-format-for-video